apps/documentation/src/routes/documentation/rendering/server-side-rendering.mdx

```diff
@@ -9,6 +9,50 @@ import Breadcrumbs from '@/components/Breadcrumbs'
 
 <Breadcrumbs breadcrumbs={[{ label: 'Rendering' }]} />
 
-# SSR
+# Server side rendering
 
-TODO
+## Overview
+
+If a page uses **Server-side Rendering**, the page HTML is generated on **each request**.
+
+> Every page is automatically server side rendered by default.
+
+To enable SSR on a page you just need to create the same file with the rust extension:
+
+- `src/routes/about.tsx`
+- `src/routes/about.rs`
+
+and then create an `#[tuono_lib::handler]` in the rust file that will be the
+function actually called on each request before rendering the HTML.
+
+## Example
+
+For example, suppose that your page needs to pre-render frequently updated data
+(read from a local file). You can write an hanlder which reads a json file and passes
+it to the route like below:
+
+```rs
+// src/routes/about.rs
+use tuono_lib::{Request, Response, Props};
+use std::fs;
+
+#[tuono_lib::handler]
+async fn my_custom_ssr_page(req: Request) -> Response {
+  let data = fs::read_to_string("my_data.json").expect("Failed to read json file");
+
+  Response::Props(Props::new(data))
+}
+```
+
+Then you will be able to read the loaded data on the route in the route's `data` props:
+
+```tsx
+// src/routes/about.tsx
+import type { TuonoProps } from 'tuono'
```

Comment:
I already bring this up in #450, but IMHO we should give a more semantic name to this type.

(We can address this in a different PR)